packages/components/text/src/Text.types.ts

```diff
@@ -1,20 +1,21 @@
-import { ITextTokens, IForegroundColorTokens } from '@fluentui-react-native/tokens';
-import { ITextProps as INativeTextProps } from '@fluentui-react-native/adapters';
-
-export const textName = 'RNFText';
-
-/**
- * Properties for fabric native text field, these extend the default props for text
- */
-export type ITextProps<TBase = INativeTextProps> = TBase &
-  ITextTokens &
-  IForegroundColorTokens & {
-    disabled?: boolean;
-  };
-
-export type ITextType<TBase = INativeTextProps> = {
-  props: ITextProps<TBase>;
-  slotProps: {
-    root: TBase;
-  };
-};
+import { ITextTokens, ITextVariantTokens, IForegroundColorTokens, IColorTokens } from '@fluentui-react-native/tokens';
+import { ITextProps as INativeTextProps } from '@fluentui-react-native/adapters';
+
+export const textName = 'RNFText';
+
+/**
+ * Properties for fabric native text field, these extend the default props for text
+ */
+export type ITextProps<TBase = INativeTextProps> = TBase &
+  ITextVariantTokens &
+  IForegroundColorTokens & {
+    disabled?: boolean;
+  };
+
+export type ITextType<TBase = INativeTextProps> = {
+  props: ITextProps<TBase>;
+  tokens: ITextTokens & IColorTokens;

```

Comment:
not sure why these two tokens are here and those part of ITextProps are different; what am I missing? :)

Reply:
Seems to me like we're only enabling a subset of tokens to be props on Text to push people to customize text just with those props and not the full set of tokens.